src/settings/Settings.tsx

```diff
@@ -123,22 +153,39 @@ export default function Settings() {
     ]
   };
 
+  const bufferStoragePathPlaceholder = state.bufferStoragePath ? state.bufferStoragePath : "(Default)";
+
   return (
     <div className="container">
+      <InformationDialog
+        title='Invalid Configuration'
+        open={dialogState.open}
+        buttons={['ok']}
+        onClose={closeDialog}
+      >
+        {dialogState.dialogContent}
+      </InformationDialog>
+
       <div className="col-xl-9 col-lg-9 col-md-12 col-sm-12 col-12">
         <div className="card h-100">
           <div className="card-body">
             <div className="row gutters">
               <div className="col-xl-6 col-lg-6 col-md-6 col-sm-6 col-12">
                 <div className="form-group">
                   <label> Storage Path </label>
-                  <input type="text" className="form-control" id="storage-path" placeholder={state.storagePath} onClick={openStoragePathDialog}/>
+                  <input type="text" className="form-control" id="storage-path" placeholder={state.storagePath} onClick={() => openDirectorySelectorDialog('storagePath')}/>
+                </div>
+              </div>
+              <div className="col-xl-6 col-lg-6 col-md-6 col-sm-6 col-12">
```

Comment:
I think this is too much power for a regular user to be able to set this - not suggesting you change this but I think it has potential to be confusing.

Will be good to move to an advanced settings tab. Hopefully I can find time to re-do the settings UI a bit this weekend.

Reply:
I was under the impression that this was a thing we'd want now? You don't have to mege this now if you don't want this setting in the app at this point. If you'd rather wait until the Settings has been revamped with an _Advanced_ tab, that's fine by me - I can keep the branch alive and make a new PR when the time is right - what do think?

Reply:
Nah lets get it in, less PRs hanging over my head 😄